docs/docs/20-usage/20-workflow-syntax.md

```diff
@@ -502,17 +502,27 @@ steps:
 
 :::
 
-### `volumes`
+### `needs`
 
-Woodpecker gives the ability to define Docker volumes in the YAML. You can use this parameter to mount files or folders on the host machine into your containers.
+Just like [`depends_on`](#depends_on), but for services.
```

Comment:
Why new keyword? Why not `depends_on`?

Reply:
What is the probability, that a service (noun like postgres, mongo) would have name like a step (verb like prepare, build, test, build-image, push-image)? And in those rare cases, when they do, we can throw error like `ambigous names in depends_on, rename step or service`.
If this is the only reason, I propose to use only `depends_on`.

Reply:
Clean your pipeline: rename steps to `test-mysql`, `test-postgres` or (`unit-test`, `build`) `integration-test-sqlite`, `integration-test-mysql`, `integration-test-postgres`. No need in different keyword.
Let me ask some questions: what is the behavior, if I define multiple steps with the same name? What if I define multiple services with the same name? Apply this behavior to inter-group naming.
If I can make both of them with the same names _and_ you do not want throwing error on the same names when `depends_on` is used, alright - do not fail, but depend on all of them. Except on step itself, which defines `depends_on`, obviously. Again, no need in separate keyword, IMO.